draft-nygren-dnsop-svcb-httpssvc.md

```diff
@@ -627,6 +627,20 @@ in cache before performing any followup queries.
 With these optimizations in place, and conforming DNS servers,
 using SVCB does not add network latency to connection setup.
 
+Moreover, if an A or AAAA response arrives before an SVCB response, a client which 
+prefers ESNI SHOULD wait up to CD milliseconds before starting secure connections 
+to either address. (Clients may begin TCP connections in this time. QUIC connections 
+should wait. This allows receipt of latent keying material in a SVCR repsonse.) 
+If an SVCB record with a "esnikeys" value and "ipv4hint" or "ipv6hint" values 
+arrives and one of those hints matches an in-progress connection, the secure 
```

Comment:
This should also reference {#optimizations} and also allow the use of responses in the client DNS cache, not just hints.  In-particular due to this it may often be possible that the hint values aren't needed if the SvcDomainName already has valid A/AAAA records in-cache (eg, due to being part of the CNAME chain from the A/AAAA lookups issued in-parallel):

> To avoid a delay for clients using a nonconforming recursive resolver, domain owners SHOULD use a single SVCB record whose SvcDomainName is in the origin hostname's CNAME chain if possible. This will ensure that the required address records are already present in the client's DNS cache as part of the responses to the address queries that were issued in parallel.

Reply:
Yes, great point! I'll add that reference, and specify that cached answers are viable.